src/libs/actions/Session.js

```diff
@@ -86,7 +86,9 @@ function fetchAccountDetails(login) {
     API.GetAccountStatus({email: login})
         .then((response) => {
             if (response.jsonCode === 200) {
-                Onyx.merge(ONYXKEYS.CREDENTIALS, {login});
+                Onyx.merge(ONYXKEYS.CREDENTIALS, {
+                    login: response.normalizedLogin ?? login,
```

Comment:
If we have `jsconCode === 200` then seems like there is no reason to fallback on `login`.

Separately, I don't think we have standardized on using the nullish coalescing operator yet...

Reply:
I changed it to include `?? login` because otherwise a couple of Jest tests were failing.

Reply:
It was a little unclear to me how to get the test to return a `normalizedLogin` in the response from the API but if you have a suggestion I'm happy to try it out further.

Reply:
Oh hmm yeah sorry it is not very obvious 😓  

We _should_ be able to fix this by passing the `normalizedLogin` as a param here under `requiresTwoFactorAuth` as this is mocking the response of `GetAccountStatus`.

https://github.com/Expensify/Expensify.cash/blob/f12fe859497b756344797f15d5e9a1ca7ee8cf94/tests/utils/TestHelper.js#L18-L23

Reply:
Oh - that actually seems super obvious now that you've pointed it out 😅